app/serializers/api/admin/order_serializer.rb

```diff
@@ -16,7 +61,33 @@ def phone
     object.billing_address.nil? ? "a" : ( object.billing_address.phone || "" )
   end
 
+  def created_at
+    object.created_at.blank? ? "" : I18n.l(object.created_at, format: '%B %d, %Y')
+  end
+
   def completed_at
-    object.completed_at.blank? ? "" : object.completed_at.strftime("%F %T")
+    object.completed_at.blank? ? "" : I18n.l(object.completed_at, format: '%B %d, %Y')
+  end
+
+  def pending_payments
```

Comment:
from here to the bottom it's biz logic, it needs to go to a service or to the payment model. right?